client/src/assets/player/peertube-player-manager.ts

```diff
@@ -284,8 +291,16 @@ export class PeertubePlayerManager {
       poster: commonOptions.poster,
       inactivityTimeout: commonOptions.inactivityTimeout,
       playbackRates: [ 0.5, 0.75, 1, 1.25, 1.5, 1.75, 2 ],
-
+      techOrder: [ 'chromecast', 'html5' ],
       plugins,
+      
+      chromecast: {
+        modifyLoadRequestFn: function (loadRequest: any) {
+          loadRequest.media.hlsSegmentFormat = 'fmp4';
+          loadRequest.media.hlsVideoSegmentFormat = 'fmp4';
+          return loadRequest;
+        }
+      },
```

Comment:
Borde inte detta ligga under `plugins`?

Reply:
videojs-chromecast consists of a plugin, a "tech", and the button component. This here is the `tech` options. The LoadRequest happens in the tech, so this belongs here. But a bit confusing, I agree.

https://github.com/andreasgan/videojs-chromecast-peertube/blob/72d6b082a86ac655c43d435d1ffb42e4042de455/src/js/enableChromecast.js#L152